.github/workflows/pull-request.yml

```diff
@@ -5,14 +5,12 @@ on:
 
 jobs:
   build:
+    name: PHP ${{ matrix.php-versions }} Test
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         operating-system: [ ubuntu-20.04 ]
-        php-versions: ['8.1']
-
-    name: PHP ${{ matrix.php-versions }} Test
+        php-versions: ['8.1', '8.2', '8.3']
```

Comment:
```suggestion
        php-versions: ['8.1', '8.3']
```

Reply:
Why is `8.2` not necessary? Is this considered overkill?

Reply:
Jep